openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -105,6 +105,7 @@ The standards that are being profiled in this specification are:
 * ISO/IEC 18013-5:2021 Personal identification — ISO-compliant driving licence Part 5: Mobile driving licence (mDL) application [@!ISO.18013-5]
 
 Note that these standards in turn build upon other underlying standards, and requirements in those underlying standards also need to be followed.
+One notable such standard is DPoP [@!RFC9449], which, through its inclusion in OpenID for Verifiable Credential Issuance, requires Wallets to be prepared to handle the `DPoP-Nonce` HTTP response header from the Credential Issuer’s Nonce Endpoint, as well as from other applicable endpoints of the Credential Issuer and Authorization Server. 
```

Comment:
I believe this should be moved into the OpenID4VCI section

Reply:
I think the proposed text itself looks good. agree with Paul that there might be a better location for it

Reply:
+1

Reply:
The the least awkward place I could find at the time to put it doesn't make it the best place. I'll find a different awkward place.

Reply:
a3084d9d2b40174412f097648fbaa812f0e0095a moved it